charts/aws-fluent-bit/values.yaml

```diff
@@ -26,7 +34,8 @@ readinessProbe:
     path: /api/v1/health
```

Comment:
could you confirm me if the api endpoint is ok. I get the connection refused

Reply:
see: https://docs.fluentbit.io/manual/administration/monitoring#health-check-for-fluent-bit

Looks fine to me as long as it is enabled and the port is open.